pkg/web/web.go

```diff
@@ -31,6 +32,11 @@ func RespondOK(ginCtx *gin.Context, data interface{}) {
 	ginCtx.Writer.Write(bytes)
 }
 
+func RespondOKWithoutData(ginCtx *gin.Context) {
+	ginCtx.Header("Content-Type", "application/json; charset=utf-8")
+	ginCtx.Status(http.StatusOK)
```

Comment:
Can make it [204 No Content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204)